PyTDF/TDataFrame.py

```diff
@@ -17,16 +17,5 @@ def __init__(self, treename, filelist):
         super(TDataFrame, self).__init__(None, None)
         self.filelist = filelist
         self.treename = treename
-
-    def create_tdf(self):
-        """
-        Method to create a TDataFrame object
-
-        """
-
-        self.Tchain = ROOT.TChain(self.treename)
-
-        for f in self.filelist:
-            self.Tchain.Add(f)
-
-        self._tdf = ROOT.Experimental.TDataFrame(self.Tchain)
\ No newline at end of file
+        self.TChain = None
+        self.TDF = None
```

Comment:
do we still need these two variables?

Reply:
@bluehood , yes. I need to store both those variables. Otherwise, it'd throw the error that we discussed yesterday. Also, storing both of them will keep them in scope

Reply:
this is my understanding: the c++ TDF needs to be in scope _until the end of the event loop_. since the event loop should run in the backend, the TDF only needs to be in scope in the backend. what am I missing?